analytics/src/main/java/brocoeur/example/analytics/service/UserWinLossByGameService.java

```diff
@@ -51,27 +51,18 @@ private void updateAnalyticAccordingToWinOrLoss(final PlayerResponse playerRespo
             throw new IllegalStateException("UserWinLossByGame with gameId : " + gameId + " and userId : " + userId + " do not exists.");
         }
 
-        final List<Boolean> listOfIsWinner = playerResponse.getListOfIsWinner();
-
-        int updatedNumberOfWin = userWinLossByGame.getNumberOfWin();
-        int updatedNumberOfLoss = userWinLossByGame.getNumberOfLoss();
-        for (Boolean aBoolean : listOfIsWinner) {
-            if (Boolean.TRUE.equals(aBoolean)) {
-                updatedNumberOfWin += 1;
-            } else {
-                updatedNumberOfLoss += 1;
-            }
-        }
+        final int amountOfMoneyWon = playerResponse.getFinalAmount() - playerResponse.getInitialAmount();
+        final int totalAmountOfMoneyWon = userWinLossByGame.getAmountOfMoneyWon();
+        final int newTotalAmountOfMoneyWon = totalAmountOfMoneyWon + amountOfMoneyWon;
```

Comment:
Why having two variables named basically the same but representing different things ? 
+ the above variable is "amountOfMoneyWon" which is what you just get from userWinLossByGame 
These variables name aren't clear enough to understand at first sight

Reply:
I agree, var name and maybe cass table element should be reviewed. Will do in next mr.